mcs/class/System/System.Security.Cryptography.X509Certificates/X509Certificate2ImplMono.cs

```diff
@@ -62,6 +62,8 @@ internal class X509Certificate2ImplMono : X509Certificate2Impl
 
```

Comment:
I am not sure whether it's a good idea to make changes to this class as it is going to be retired once we have removed the Legacy TLS Provider.

Adding new features to it is also a bit problematic in the sense that it would commit us to also supporting these in the new implementation.

Reply:
The change is moving code from internal X509ExtensionCollection constructor directly to the only place where it was used. The constructor is not present in CoreFX code.

Reply:
It's fine to move it here to simplify the migration

Reply:
I have essentially completely replaced the `X509Certificate2ImplMono` class in my latest working tree.